### PR TITLE
Move Platinum Service Worker support to a recipe

### LINKED ISSUE #411: Move Platinum Service Worker support to a recipe
We've discussed this a few times in recent meetings. The goal is to allow PSK to trim down to a leaner base and make everything currently optional a more streamlined recipe. Still open for discussion :speaker:

### PR DESCRIPTION
**Pull request by addyosmani**
Fixes #411. r: @robdodson @kaycebasques @samccone

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,103 +150,7 @@ Polymer uses [Bower](http://bower.io) for package management. This makes it easy
...
-
-#### Disable Service Worker support after you enabled it
-
-If for any reason you need to disable Service Worker support after previously enabling it, you can remove it from your Polymer Starter Kit project using these 4 steps:
-
-1. Remove references to the platinum-sw elements from your application [index](https://github.com/PolymerElements/polymer-starter-kit/blob/master/app/index.html).
-2. Remove the two Platinum Service Worker elements (platinum-sw/..) in [app/elements/elements.html](https://github.com/PolymerElements/polymer-starter-kit/blob/master/app/elements/elements.html)
-3. Remove 'precache' from the list in the 'default' gulp task ([gulpfile.js](https://github.com/PolymerElements/polymer-starter-kit/blob/master/gulpfile.js))
-4. Navigate to `chrome://serviceworker-internals` and unregister any Service Workers registered by Polymer Starter Kit for your app just in case there's a copy of it cached.
+Polymer Starter Kit offers an optional offline experience thanks to Service Worker and the [Platinum Service Worker elements](https://github.com/PolymerElements/platinum-sw). See our [recipe]() for the complete walkthrough.
```
Comment (chuckh) [inline README.md]: `[recipe]()` needs a link like `[recipe](https://github.com/PolymerElements/polymer-starter-kit/blob/master/docs/offline-service-worker.md)`

**Comment by chuckh** [comment]
LGTM except for my line 153 comment.

**Comment by robdodson** [comment]
this makes me a bit sad. I'm concerned folks won't enable it now and it's seriously awesome :'(

**Comment by addyosmani** [comment]
@robdodson we don't necessarily have to move ahead with this. Offline-support is something of a product decision and I'd love to hear what @tjsavage thinks about this moving to a recipe being commented out. @samccone had thoughts about this (iirc) in our last meeting.

**Comment by addyosmani** [comment]
friendly ping @tjsavage

**Comment by addyosmani** [comment]
After considering this further, I think it's worth keeping in support for Platinum as is using the comment-in model. Rob's point about it being unlikely folks will switch to using it once in a recipe is fair and I'd rather us encourage more Service Worker-enabled offline apps now that https://letsencrypt.org/ is a real option.

**Pull request closed without merging**